src/views/dashboard/related/trace/Filter.vue

```diff
@@ -100,8 +110,8 @@ const selectorStore = useSelectorStore();
 const dashboardStore = useDashboardStore();
 const traceStore = useTraceStore();
 const traceId = ref<string>("");
-const minTraceDuration = ref<string>("");
-const maxTraceDuration = ref<string>("");
+const minTraceDuration = ref<number>(NaN);
+const maxTraceDuration = ref<number>(NaN);
```

Comment:
Is this for the duration query bug fix? If so, update the change log when sync to the main repo.

Reply:
Yes. I will do.